makefile

```diff
@@ -182,6 +182,7 @@ clean-manual:
 
 clean-all: clean clean-deps clean-libraries clean-manual
 	$(RM) -r bin
+	$(RM) -r src/cmdstan/main.o
```

Comment:
You want to delete `$(CMDSTAN_MAIN_O)` here and not directly name the file defined in the variable.

Reply:
Can this be adjusted?